supervisor/mounts/manager.py

```diff
@@ -141,6 +141,15 @@ async def load(self) -> None:
                 ]
             )
 
+        # Bind all backup mounts to directories in backup
```

Comment:
Ah I see. I am a bit worried, so now we need to maintain another location where network storage are bind mounted :cold_sweat: 

Hm, why didn't we do that in first place?

Reply:
Mostly just because we didn't need to. There wasn't any reason to expose backup mounts to others because supervisor was doing all the operations on them. I'm not exactly sure how addons like Google Drive sync, etc. will handle it though, it depends on how they process the `/backup` folder.

The only other options I could think of was bind mounting the entire `/mounts` folder to supervisor into core's container. This would work but it seemed worse for a few reasons:

1. All the share and media mounts are available in multiple places in core's container now
2. A user could now muck with this container in an automation/script and really mess up supervisor (like if they deleted the folder)
3. I don't know if we'll have other types of mounts in the future we want to keep more hidden

Exposing addons to backup network mounts felt less risky. I couldn't think of any reason we didn't do it to begin with other then "we didn't have to".

Reply:
> I'm not exactly sure how addons like Google Drive sync, etc. will handle it though, it depends on how they process the `/backup` folder.

Hm, I see, that could actually have potential implications, e.g. if they browse often etc. it will lead to network traffic etc.

Instead of going to a separate mount, could we optionally mount without mount propagation maybe? :thinking:

Reply:
Just for anyone else interested - tried this, did not work as we hoped. When propagation is set to `rprivate` the addons can still see everything in the network mounts. Its just changes to backup mounts require a restart of the addon. So it really just makes it inconsistent, doesn't help.